R/package.R

```diff
@@ -124,6 +124,29 @@ initialize_python <- function(required_module = NULL, use_environment = NULL) {
   # ensure modules can be imported from the current working directory
   py_run_string_impl("import sys; sys.path.insert(0, '')")
 
+  # notify the user if the loaded version of Python isn't the same
+  # as the requested version of python
+  local({
+
+    requested_versions <- reticulate_python_versions()
+    if (length(requested_versions) == 0)
+      return(TRUE)
```

Comment:
Are these just early returns? (i.e. does the `TRUE` do anything here?)

Reply:
Just early returns (no meaning for return value)